File: LEETCODE/面试题03. 数组中重复的数字.java
class Solution {
    public int findRepeatNumber(int[] nums) {
        Map<Integer, Integer> map = new HashMap<>();
        for(int num: nums){
            if(map.containsKey(num)){
                return num;
            }else{
                map.put(num, 1);
            }
        }
        return -1;
    }
}

File: LEETCODE/kotlin/面试题03. 数组中重复的数字.kt
class Solution {
    fun findRepeatNumber(nums: IntArray): Int {
        var map: MutableMap<Int, Int> = mutableMapOf()
        for(num in nums) {
            if(map.contains(num)){
                return num
            }else{
                map.put(num, 1)
            }
        }
        return -1
    }
}